### PR TITLE
added routing, navbar, footer

### PR DESCRIPTION
**Pull request by muskanbagrecha**
Added routing, navbar and footer. 
Files changed:
Footer: src/Components/UI/Footer/Footer.jsx
Navigation: src/Components/UI/Navigation/Navigation.jsx
Router: src/Router/AppRouter.jsx

### CODE REVIEW

File: src/Components/UI/Navigation/Navigation.jsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      <div
+        className={`burger ${burgerRotate}`}
+        onClick={() => {
+          setMobileNavActive((prevState) => !prevState);
+        }}
+      >
+        <div className="line1"></div>
+        <div className="line2"></div>
+        <div className="line3"></div>
+      </div>
```
Comment (aiman-ansari) [inline src/Components/UI/Navigation/Navigation.jsx:85]: Great work !!
Saw something new to make it responsive .

File: src/Pages/Homepage/Homepage.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { Navigation, Footer } from "../../Components/UI/";
```
Comment (aiman-ansari) [inline src/Pages/Homepage/Homepage.jsx]: Why you import these here?

**Review by aiman-ansari — APPROVED** [review]
- Neat and clean code .
- Your UI looks superb clean and cool.
- Like the way you make it responsive.
- declarative variable naming everything looks good to proceed.

File: src/Assets/Icons/icons.jsx [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
-      <svg width="1em" height="1em" viewBox="0 0 24 24" {...props}><path fill="currentColor" d="m12 21.35l-1.45-1.32C5.4 15.36 2 12.27 2 8.5C2 5.41 4.42 3 7.5 3c1.74 0 3.41.81 4.5 2.08C13.09 3.81 14.76 3 16.5 3C19.58 3 22 5.41 22 8.5c0 3.77-3.4 6.86-8.55 11.53L12 21.35Z"></path></svg>
+      <svg width="1.25em" height="1.5em" viewBox="0 0 24 24" {...props}><path fill="currentColor" d="m12 21.35l-1.45-1.32C5.4 15.36 2 12.27 2 8.5C2 5.41 4.42 3 7.5 3c1.74 0 3.41.81 4.5 2.08C13.09 3.81 14.76 3 16.5 3C19.58 3 22 5.41 22 8.5c0 3.77-3.4 6.86-8.55 11.53L12 21.35Z"></path></svg>
     )
   }
 
 
 function User(props) {
     return (
-      <svg width="1em" height="1em" viewBox="0 0 24 24" {...props}><path fill="currentColor" d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4s-4 1.79-4 4s1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"></path></svg>
+      <svg width="1.5em" height="1.5em" viewBox="0 0 24 24" {...props}><path fill="currentColor" d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4s-4 1.79-4 4s1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"></path></svg>
```
Comment (m26unkwn) [inline src/Assets/Icons/icons.jsx]: You can create .svg file for this i think.

File: src/Pages/NotFound.jsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import { useNavigate } from "react-router-dom";
+
+const NotFound = (props) => {
+  const navigate = useNavigate();
+
+  return (
+    <div className="sub-container">
+      <h1>404</h1>
+      <p>Uh Oh, Page Not Found! :(</p>
+      <button onClick={() => {navigate("/")}} className="btn btn-primary margin-top-2">
```
Comment (m26unkwn) [inline src/Pages/NotFound.jsx]: Hey you can use single line code here as you know arrow function support this.
`()=>navigate("/")`

**Review by m26unkwn — APPROVED** [review]

File: src/Pages/Homepage/Homepage.jsx (hunk 2 above)
Comment (muskanbagrecha) [reply src/Pages/Homepage/Homepage.jsx]: Forgot to remove it. Thanks for pointing it out.

**Pull request merged**